cmake/packaging.cmake

```diff
@@ -250,12 +252,23 @@ configure_file(cmake/CPackConfig.cmake.in package/CPackConfig.cmake @ONLY)
 set(CPACK_GENERATOR "${PACKAGING_SYSTEM}")
 
 if(DEFINED PLATFORM_LINUX)
+  set(CPACK_PACKAGING_INSTALL_PREFIX "/")
```

Comment:
Head ups, if you look at the above changes the `/etc/default` absolute values are changed to relative values `etc/default`. I can remove this line `set(CPACK_PACKAGING_INSTALL_PREFIX "/")` and restore the absolute values if that is best?

Reply:
I added this line mostly to solve for TGZ building. With the absolute values the build fails as it tries to install to those absolute locations during `make package`. I can try to implement your suggestion of DESTDIR settings later today.

Reply:
> Head ups, if you look at the above changes the `/etc/default` absolute values are changed to relative values `etc/default`. I can remove this line `set(CPACK_PACKAGING_INSTALL_PREFIX "/")` and restore the absolute values if that is best?

Yes, this is idiomatic in CMake in general, you use CMAKE_INSTALL_PREFIX to set the prefix to install, which is `/usr` by default, and always use relative paths if you want the prefix to be applied; If not you use absolute paths.
Then If you really have to relocate everything, you set the DESTDIR env variable when you install, then that value will be the root; relative values will still have a their prefix.
This is what CPack internally uses to install the project in its own folder, which later processes to create the package.

 > I added this line mostly to solve for TGZ building. With the absolute values the build fails as it tries to install to those absolute locations during make package. I can try to implement your suggestion of DESTDIR settings later today.

CPACK_SET_DESTDIR is definitely the solution there, you just need to set it ON.